ui-backend/catalog-ui-search/pom.xml

```diff
@@ -296,6 +296,7 @@
         </dependency>
         <dependency>
             <groupId>ddf.platform.security</groupId>
+            <!-- TODO rename to session-management-api when upgrading to DDF 2.25.x -->
```

Comment:
I think you should do the rename and upgrade the ddf version. The PR will be on hold until DDF is released

Reply:
Thanks for the advice 😄